Gemfile

```diff
@@ -1,10 +1,11 @@
 # frozen_string_literal: true
 
-source "https://rubygems.org"
+source 'https://rubygems.org'
 
-git_source(:github) {|repo_name| "https://github.com/#{repo_name}" }
+git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
 
-gem 'rspec'
-gem 'guard-rspec'
 gem 'guard'
+gem 'guard-rspec'
```

Comment:
No need to have `guard` as well as `guard-rspec`. The latter is fine on its own.